### PR TITLE
Updated scripts to include Data

### PR DESCRIPTION
**Pull request by jackycodes**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Updated scripts to include Data

### CODE REVIEW

File: scripts/bump-ios-sdk-version.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ fi
 echo "React-Native iOS version $oldiOSSdkVersion will be updated to $newiOSSdkVersion"
 
 # Update iOS sdk version in postlink.js for appcenter-crashes, appcenter-analytics,
-# appcenter-auth appcenter-push projects
+# appcenter-auth appcenter-push projects appcenter-data
```
Comment (dhei) [inline scripts/bump-ios-sdk-version.sh]:
```suggestion
# appcenter-auth appcenter-push appcenter-data projects
```

File: scripts/bump-wrapper-sdk-version.sh [hunk 2]
```diff
@@ -51,23 +51,25 @@ echo "React-Native SDK $oldWrapperSdkVersion will be updated to $newWrapperSdkVe
...
 cat ./appcenter/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."appcenter-link-scripts" = env.newVersion' > ./appcenter/package.json.temp && mv ./appcenter/package.json.temp ./appcenter/package.json
 cat ./appcenter-crashes/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."appcenter" = env.newVersion' > ./appcenter-crashes/package.json.temp && mv ./appcenter-crashes/package.json.temp ./appcenter-crashes/package.json 
 cat ./appcenter-analytics/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."appcenter" = env.newVersion' > ./appcenter-analytics/package.json.temp && mv ./appcenter-analytics/package.json.temp ./appcenter-analytics/package.json
 cat ./appcenter-auth/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."appcenter" = env.newVersion' > ./appcenter-auth/package.json.temp && mv ./appcenter-auth/package.json.temp ./appcenter-auth/package.json
+cat ./appcenter-data/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."appcenter" = env.newVersion' > ./appcenter-data/package.json.temp && mv ./appcenter-data/package.json.temp ./appcenter-auth/package.json
```
Comment (dhei) [inline scripts/bump-wrapper-sdk-version.sh]:
```suggestion
cat ./appcenter-data/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."appcenter" = env.newVersion' > ./appcenter-data/package.json.temp && mv ./appcenter-data/package.json.temp ./appcenter-data/package.json
```

File: TestApp/copy-code-changes-to-git.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 echo 'Copy SDK code changes made inside node_modules to git...'
 cd `dirname $0`
-for i in '' '-analytics' '-crashes' '-push'
+for i in '' '-analytics' '-crashes' '-push' '-data'
```
Comment (dhei) [inline TestApp/copy-code-changes-to-git.sh]: Looks like we forgot `-auth`. Let's add it.

Comment (jackycodes) [reply TestApp/copy-code-changes-to-git.sh]: `-auth` was added in another [PR](https://github.com/microsoft/appcenter-sdk-react-native/pull/638/files#diff-7dd8fda13620c96a2fc8ad5978c44903L4). It’s in the develop branch: [https://github.com/microsoft/appcenter-sdk-react-native/blob/25163fe18224dc502547c594bb3ccaeb0784802d/TestApp/copy-code-changes-to-git.sh#L4](https://github.com/microsoft/appcenter-sdk-react-native/blob/25163fe18224dc502547c594bb3ccaeb0784802d/TestApp/copy-code-changes-to-git.sh#L4)

File: scripts/bump-ios-sdk-version.sh (hunk 1 above)
Comment (jackycodes) [reply scripts/bump-ios-sdk-version.sh]: fixed

File: scripts/bump-wrapper-sdk-version.sh (hunk 2 above)
Comment (jackycodes) [reply scripts/bump-wrapper-sdk-version.sh]: fixed

File: TestApp/copy-code-changes-to-git.sh (hunk 3 above)
Comment (dhei) [reply TestApp/copy-code-changes-to-git.sh]: Got it, thanks.

**Pull request merged**